q2_demux/tests/test_demux.py

```diff
@@ -863,7 +863,7 @@ def test_basic(self):
             index_fp = os.path.join(output_dir, 'overview.html')
             self.assertTrue(os.path.exists(index_fp))
             self.assertTrue(os.path.getsize(index_fp) > 0)
-            # NEW TODO: This doesn't get created anymore
+            # NEW TODO: This file is talked about on line 221, but never created?
```

Comment:
I think it _is_ being created, its just a different extension (TSV in the viz code, but CSV here in the test) - I would start with updating the extensions here in the test suite.

I changed this to TSV (from CSV) because we have been working towards "standardizing" that format across our plugins.

Reply:
Yeah I realized a moment ago that I had misread some parts of the changed code last night which prompted that sort of roleback commit. Things are making a lot more sense now.